libs/ledger-live-common/src/postOnboarding/hooks/useClearLastActionCompletedCallback.ts

```diff
@@ -0,0 +1,10 @@
+import { useCallback } from "react";
```

Comment:
I'm a bit in doubt about hiding all this simple logic under hooks.
I understand the idea of having it here since it will be used by both LLD and LLC. But it has a cost, indirection.
Every time someone wants to understand what this hook is really doing they'll have to open it, while if it was directly in the component they would know that it dispatches an action.

Redux itself already creates some indirection since we have to check the id of the action and then look for the reducer that handles it. Than there's more this indirection. I don't know. Sometimes I feel that DRY gets a bit overrated.

Just a thought not really saying that we need to change anything here.